rpc/src/rpc_subscriptions.rs

```diff
@@ -416,14 +465,8 @@ fn initial_last_notified_slot(
                 0
             }
         }
-        // last_notified_slot is not utilized for these subscriptions
-        SubscriptionParams::Logs(_)
-        | SubscriptionParams::Program(_)
-        | SubscriptionParams::Signature(_)
-        | SubscriptionParams::Slot
-        | SubscriptionParams::SlotsUpdates
-        | SubscriptionParams::Root
-        | SubscriptionParams::Vote => 0,
+        // last_notified_slot is not utilized for other subscription types
```

Comment:
This PR does use last_notified_slot for SubscriptionParams::Block, though, doesn't it?

Reply:
hm yea not sure about this comment, it's from a previous commit. but with regards to `SubscriptionParams::Block`, it does use this field... I've rm'd the comment